### ISSUE TITLE
No lock screen on chroot startup

### ISSUE THREAD

**Issue by kingliamb** [dnschneid/crouton#4236]
`$ sudo edit-chroot -all`
> `name: thistlehoney`
> `encrypted: no`
> `Entering /mnt/stateful_partition/crouton/chroots/thistlehoney...`
> `crouton: version 1-20200116214215~master:9017d640`
> `release: xenial`
> `architecture: amd64`
> `xmethod: xorg`
> `targets: unity-desktop,xorg,xiwi`
> `host: version 12871.34.0 (Official Build) beta-channel coral`
> `kernel: Linux localhost 4.4.211-17198-ga431862f6e61 #1 SMP PREEMPT` *the date was edited out here*
> `2020 x86_64 x86_64 x86_64 GNU/Linux`
> `freon: yes`
> `Not unmounting /mnt/stateful_partition/crouton/chroots/thistlehoney as another instance is using it.`

#### Issue
Ubuntu Xenial with Unity (or any chroot os/desktop combination I've tried for that matter) doesn't lock (display the lock screen which requires a password) when I boot into it. I have to manually press **Search + l** to lock it. I'm using the automatic boot found [here](https://github.com/dnschneid/crouton/wiki/Autostart-crouton-chroot-at-ChromeOS-startup), so when I start my computer, it goes straight into the chroot, which is a huge physical security problem. Is there a way to do this? There isn't anything anywhere on the web.

#### Reproduce steps
`$ sudo startunity` or really any `startwhatever` for Ubuntu and Debian (I haven't tried Kali).

#### Additional Notes
To be clear, when I say "lock screen", this is what I am referring to. 
> ![Lockscreen](https://user-images.githubusercontent.com/55339220/76659040-d0e76680-654b-11ea-9f88-eedb7f17619a.png)

Thanks for your help!

**Comment by DennisLfromGA** [comment]
@kingliamb,

There's no Desktop manager like lightdm, etc. in a crouton chroot so you won't get the screen you show above to lock it.

Perhaps you could add a screensaver app like `gnome-screensaver` and then use someting like the below to lock it:
- `gnome-screensaver-command -l`

*Hope this helps,*
-**DennisLfromGA**

**Comment by kingliamb** [comment]
So, I tried looking for some way to run scripts from startup in Linux. On windows the Startup folder accepts `.bat` and `.exe` files, but it doesn't look like there's an already setup folder like that on Debian.
It looks like there's a script `/etc/rc.local` that, by default, does nothing, but can be configured to run basically any command. I added `gnome-screensaver-command -l` between all the hashtags and `exit 0`. For people seeing this issue that don't know how to do that, here's a quick tutorial.

A `$` indicates a bash command prompt. Don't actually paste that into the terminal; you'll get an error.
Open a terminal (I used gnome-terminal)
`$ sudo setsid gedit /etc/rc.local`
Add `gnome-screensaver-command -l` between all the hashtags and `exit 0`
* Note: It is imperative that the script ends with `exit 0`. Be sure to leave it in.
Save the file with `Ctrl + s` and close the window. Reboot your system to make sure it works.
Thanks for the help @DennisLfromGA
